src/rust/ide/lib/view/src/graph_editor/component/breadcrumbs/breadcrumb.rs

```diff
@@ -0,0 +1,416 @@
+//! This module provides a clickable view for a single breadcrumb.
```

Comment:
Changing the `ICON_RADIUS` var doesn't work - the icons are not vertically centered anymore:
    ![image](https://user-images.githubusercontent.com/1623053/89508026-9c5f9a00-d7cd-11ea-9678-ff95ee9ef0f9.png)